src/tactic/tauto.lean

```diff
@@ -208,6 +209,8 @@ meta structure tauto_cfg :=
 (classical : bool                        := ff)
 (closer : tactic unit                  := pure ())
 
+attribute [nolint doc_blame] tauto_cfg
```

Comment:
Why no doc on `tauto_cfg`?

Reply:
It just bundles arguments to `tauto`, and only exists as glue between the interactive frontend and `tactic.tautology`.

But, this made me realize that `basic_tauto_tacs` should not be configurable. I added the docstring anyways.